docs/src/HOWTOs/howto-write-type-annotations.md

```diff
@@ -61,20 +65,20 @@ Consider the example [Channel.tla][] from [Specifying Systems][]:
 
 ```tla
 
-{{#include ../../../test/tla/Channel.tla::28}}
+{{#include ../../../test/tla/Channel.tla}}
```

Comment:
I just double-checked and it works for me locally. I'd consider the whole spec relevant here,
since except for an informative comment, the lines are mandated by TLA, e.g. the first line with the module and the last delimiting line of '============'